lib/miq_automation_engine/service_models/miq_ae_service_service.rb

```diff
@@ -48,6 +48,30 @@ def set_dialog_option(key, value)
       end
     end
 
+    def service_vars_options
+      @object.options[:service_vars] || HashWithIndifferentAccess.new
+    end
+
+    def get_service_vars_option(key)
+      service_vars_options[key]
+    end
+
+    def set_service_vars_option(key, value)
+      ar_method do
+        @object.options[:service_vars] ||= HashWithIndifferentAccess.new
+        @object.options[:service_vars][key] = value
+        @object.update(:options => @object.options)
+      end
+    end
+
+    def delete_service_vars_option(key)
+      ar_method do
+        @object.options[:service_vars]&.delete(key).tap do |value|
+          @object.update(:options => @object.options) if value
```

Comment:
So, we don't want to save the delete of the key if the value is `nil` or `false`?